Stripe/PublicHeaders/STPAPIResponseDecodable.h

```diff
@@ -11,6 +11,11 @@
 @protocol STPAPIResponseDecodable <NSObject>
 
 /**
+ A string representing the object's type. May be nil.
+ */
++ (nullable NSString *)object;
```

Comment:
I'm wary of adding protocol methods that are super generically named like this, for fear of conflict with other existing methods. Maybe something that makes it more stripe specific or specific to this protocol?

I'm also not sure the value of having this exposed anyway. It's helpful on web where everything is just json dicts, but on iOS if there's a json object <-> iOS class mismatch, it should just return nil from STPAPIResponseDecodeable. Not sure if we want competing instance.object and instance.class checking?